lib/widgets/action_sheet.dart

```diff
@@ -587,6 +583,32 @@ class MarkTopicAsReadButton extends ActionSheetMenuItemButton {
   }
 }
 
+class CopyTopicLinkButton extends ActionSheetMenuItemButton {
+  const CopyTopicLinkButton({
+    super.key,
+    required this.narrow,
+    required super.pageContext,
+  });
+
+  final TopicNarrow narrow;
+
+  @override IconData get icon => Icons.link;
```

Comment:
I notice that this is `Icons.link` instead of `ZulipIcons.link`. Do we have a custom icon we can use here?

…Ah I think we do. Here's the SVG: https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=544-22131&p=f&m=dev

I see it being used for "Copy link to channel" in Figma: https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=6326-96513&m=dev

Would you add a prep commit that adds that new custom icon, and another commit that changes the one existing `Icons.link` (in "Copy link to message") to use the new icon?